apps/meteor/client/views/omnichannel/directory/chats/contextualBar/AgentField.js

```diff
@@ -1,40 +1,40 @@
 import { useTranslation } from '@rocket.chat/ui-contexts';
-import React, { useMemo } from 'react';
+import React from 'react';
 
 import { UserStatus } from '../../../../../components/UserStatus';
 import UserAvatar from '../../../../../components/avatar/UserAvatar';
-import { AsyncStatePhase } from '../../../../../hooks/useAsyncState';
-import { useEndpointData } from '../../../../../hooks/useEndpointData';
 import AgentInfoDetails from '../../../components/AgentInfoDetails';
 import Field from '../../../components/Field';
 import Info from '../../../components/Info';
 import Label from '../../../components/Label';
 import { FormSkeleton } from '../../Skeleton';
+import { useUserInfo } from './hooks/useUserInfo';
 
 const AgentField = ({ agent, isSmall = false }) => {
 	const t = useTranslation();
 	const { username } = agent;
-	const { value, phase: state } = useEndpointData(
-		`/v1/users.info`,
-		useMemo(() => ({ username }), [username]),
-	);
 
-	if (state === AsyncStatePhase.LOADING) {
+	const { data, isLoading, isError } = useUserInfo(username);
+
+	if (!data || isLoading || isError) {
```

Comment:
Does it make sense to leave the skeleton when there's no data? 
Wouldn't it make more sense to not render this field at all?